docs/configuration.rst

```diff
@@ -93,22 +93,35 @@ These settings are global and have effect on both, tooltips and modal dialogues.
 
 .. confval:: hoverxref_intersphinx
 
-   Description: Whether or not enable Sphinx's hoverxref extension on intersphinx links.
+   Description: Enable Sphinx's hoverxref extension on intersphinx targets from ``intersphinx_mapping``.
 
-   Default: ``False``
+   Default: ``[]``
 
-   Type: bool
+   Type: list
 
-.. confval:: hoverxref_intersphinx_type
+   .. warning::
 
-   Description: Style used for intersphinx links.
+      The Sphinx's target project **must be hosted on Read the Docs** to work.
+      This is a current limitation that we may be able to remove in the future.
 
-   Default: ``None``. It uses :confval:`hoverxref_default_type` if not defined.
+.. confval:: hoverxref_intersphinx_types
```

Comment:
Is this useful? It seems you'd want to change the modal on content type (eg. `doc` or `class`) more than at the project-level?

Reply:
Good point!

In that case we would need a nested dictionary:

```python
hoverxref_intersphinx_types = {
 'python': {
    'ref': 'tooltip',
    'doc': 'modal',
  },
}
```

It's basically one [`hoverxref_role_types`](https://sphinx-hoverxref.readthedocs.io/en/latest/configuration.html#confval-hoverxref_role_types) per intersphinx mapping.

However, I'm not sure how easy is to detect the role used in the missing-reference. I'll take a look and come back 😄

Reply:
I found how to this. I pushed the changes already.